### PR TITLE
Add gitHubToken to prevent rate limit problems

### LINKED ISSUE #768: GitHub API rate limit exceeded when deploy server mode using helm
Hi,

Thanks for @czunker, I am trying to deploy trivy server mode in kubernetes.

In my case,

```
$ k logs -n trivy-stage trivy-stage-0 
2020-12-07T01:32:38.733Z	DEBUG	cache dir:  /home/scanner/.cache/trivy
2020-12-07T01:32:38.735Z	DEBUG	There is no valid metadata file: unable to open a file: open /home/scanner/.cache/trivy/db/metadata.json: no such file or directory
2020-12-07T01:32:38.735Z	INFO	Need to update DB
2020-12-07T01:32:38.735Z	INFO	Downloading DB...
2020-12-07T01:32:38.735Z	DEBUG	no metadata file
2020-12-07T01:32:38.880Z	FATAL	failed to download vulnerability DB:
    github.com/aquasecurity/trivy/internal/operation.DownloadDB
        /home/circleci/project/internal/operation/operation.go:77
  - failed to download vulnerability DB:
    github.com/aquasecurity/trivy/pkg/db.Client.Download
        /home/circleci/project/pkg/db/db.go:169
  - failed to list releases:
    github.com/aquasecurity/trivy/pkg/github.Client.DownloadDB
        /home/circleci/project/pkg/github/github.go:89
  - GET https://api.github.com/repos/aquasecurity/trivy-db/releases: 403 API rate limit exceeded for 1.2.3.4. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) [rate reset in 33m12s]
```

I think trivy server need to download from github for create DB first time, but in this time, user need to set github credential.

I have other questions, too.
https://github.com/aquasecurity/trivy/blob/8132174754c6aed66582ac564a1847e01e116593/helm/trivy/templates/statefulset.yaml#L56-L61
does trivy need to `vulnType` really? I think this is just DB server.
also `severity`, `ignoreUnfixed` is needed client side I think.

### PR DESCRIPTION
**Pull request by czunker**
And remove unnecessary config parameters.

Fixes #768

Signed-off-by: Christian Zunker <christian.zunker@codecentric.cloud>

### CODE REVIEW

**Comment by knqyf263** [comment]
@danielpacak As I mentioned here, I think `githubToken` is still necessary. What do you think?
https://github.com/aquasecurity/trivy/pull/751#discussion_r537220048

**Comment by Hokwang** [comment]
@czunker  OMG you are so fast~ I am creating PR too. (will remove)
Many thanks.
I want to tell you something more through inline comment.

**Comment by czunker** [comment]
@Hokwang I added your suggestions. Please have a look.

**Comment by danielpacak** [comment]
> @danielpacak As I mentioned here, I think `githubToken` is still necessary. What do you think?
> [#751 (comment)](https://github.com/aquasecurity/trivy/pull/751#discussion_r537220048)

You're right @knqyf263

File: helm/trivy/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ fullnameOverride: ""
 image:
   registry: docker.io
   repository: aquasec/trivy
-  tag: 0.13.0
+  tag: 0.14.0
```
Comment (danielpacak) [inline helm/trivy/values.yaml:7]: @czunker  Remember to bump up `appVersion` from `0.13.0` to `0.14.0` and `version` from `0.1.0` to `0.2.0` in `Chart.yaml`

Actually we could have a template function that reads the tag from the `appVersion` property, WDYT?

**Review by danielpacak — CHANGES_REQUESTED** [review]
LGTM! We just need to bump up versions in Chart.yaml to merge it!

**Comment by danielpacak** [comment]
@knqyf263 Do you think we should be publishing this Helm chart to https://helm.aquasec.com. We already publish other charts:

```
$ helm repo list
NAME   	URL
aqua   	https://helm.aquasec.com
```

```
$ helm search repo aqua
NAME                     	CHART VERSION	APP VERSION	DESCRIPTION
aqua/harbor-scanner-aqua 	0.10.1       	0.10.0     	Harbor scanner adapter for Aqua CSP scanner
aqua/enforcer            	5.3.0        	5.3        	A Helm chart for the Aqua Enforcer
aqua/harbor-scanner-trivy	0.12.0       	0.15.0     	Harbor scanner adapter for Trivy
aqua/scanner             	5.3.0        	5.3        	A Helm chart for the Aqua Scanner CLI component
aqua/server              	5.3.0        	5.3        	A Helm chart for the Aqua Console components
```

**Comment by knqyf263** [comment]
Can we publish the chart automatically on GitHub Actions?

**Comment by danielpacak** [comment]
> Can we publish the chart automatically on GitHub Actions?

Not yet, but we can think of it. Currently our internal system pull the chart from GH repo, packages and and publishes to Aqua server. But maybe we can change that if push form GH Actions runner to the Helm repo.

**Comment by knqyf263** [comment]
I see. We probably should publish the chart on GitHub like https://github.com/prometheus-community/helm-charts/tree/gh-pages.
We will not release it at this time, but if there are many requests for the registry, we'll work on it.

**Comment by Hokwang** [comment]
> @Hokwang I added your suggestions. Please have a look.

perfect.

File: helm/trivy/values.yaml (hunk 1 above)
Comment (czunker) [reply helm/trivy/values.yaml:7]: Bumped the versions.

Comment (czunker) [reply helm/trivy/values.yaml:7]: I'm not sure about the function.

**Review by danielpacak — APPROVED** [review]

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
@czunker Thanks!

**Pull request merged**